packages/feedback/src/widget/Form.ts

```diff
@@ -161,28 +167,48 @@ export function Form({
       errorEl,
 
       !isAnonymous &&
-        showName &&
+        (showName || isNameRequired) &&
         createElement(
           'label',
           {
             htmlFor: 'name',
             className: 'form__label',
           },
-          [nameLabel, nameEl],
+          isNameRequired
+            ? [
+                createElement(
+                  'span',
+                  { className: 'form__label__text' },
+                  nameLabel,
+                  createElement('span', { className: 'form__label__text--required' }, ' (required)'),
+                ),
+                nameEl,
+              ]
+            : [nameLabel, nameEl],
         ),
-      !isAnonymous && !showName && nameEl,
+      !isAnonymous && !(showName || isNameRequired) && nameEl,
```

Comment:
I would refactor this logic `showName || isNameRequired` to a var `isNameVisible` (or something like that) -- it's used quite a few different times and it describes better what it is used for (I had to read this a few times to remember what it's doing).